matrix-event.h

```diff
@@ -25,7 +25,9 @@
 #include <glib.h>
 
 struct _JsonObject;
+struct _MatrixRoomEvent;
 
+typedef void (*EventSendHook)(void *conv, struct _MatrixRoomEvent *event);
```

Comment:
Ah good spot; yep fixed; the reason I'd ended up as a void* was I was trying to avoid matrix-event.h being Purple specific since it looks like it could be split out as part of a generic C library.